aurora_module/rds_aurora_cluster.tf

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) 2011 - 2019, Coveo Solutions Inc.
+ */
+
+locals {
+  instances_count = "${var.engine_mode == "serverless" ? 0 : var.instances_count}"
+}
+
+resource "aws_db_subnet_group" "db_subnet_group" {
+  subnet_ids = "${var.subnet_ids}"
+
+  name        = "${lookup(var.optional_parameters, "db_subnet_group_name", "${var.custom_identifier}")}"
+  description = "${lookup(var.optional_parameters, "subnet_group_description", "${var.custom_identifier}")}"
+  tags        = "${var.subnet_group_tags}"
+}
+
+resource "aws_kms_key" "rds_cluster_encryption_key" {
+  tags = "${var.db_tags}"
+}
+
+resource "aws_kms_alias" "rds_cluster_encryption_key_alias" {
+  target_key_id = "${aws_kms_key.rds_cluster_encryption_key.key_id}"
+  name          = "alias/${lookup(var.optional_parameters, "rds_cluster_encryption_key_alias_name", "${var.custom_identifier}_dbek")}"
+}
+
+resource "aws_kms_key" "rds_cluster_master_passw_encryption_key" {
```

Comment:
Another KMS key? 😄 

I think you need to take a look at AWS pricing: https://aws.amazon.com/kms/pricing/

Yeah, it's only one dollar per month per key, but multiply 10 clusters x 2 keys x 12 months x $1 => $240.

Do we really need one key for the cluster and another one for SSM? :)

Reply:
I didn't see the price per key. 😅 
You as a Security Analyst, you think one key is enough for both?

Reply:
We don't need one key per cluster and another one for SSM. We are not even forced to use one key per team. We could use a shared key for multiple projects.

You can stick with "one key per development team", but don't create more than that.

Reply:
Not sure if you saw the PR update. The kms key has been removed. :P